### ISSUE TITLE
`yt-dlp` doesn't work

### ISSUE THREAD

**Issue by proninyaroslav** [Waujito/youtubeUnblock#74]
If I run `yt-dlp`, loading usually happens, but very slowly, and VPN solves this problem. But if I don't use VPN, but only  `youtubeUnblock`, `yt-dlp` gives a bunch of SSL errors like this:

```
...
 Giving up after 10 retries
[download] fragment not found; Skipping fragment 23 ...
[download] Got error: [SSL: INVALID_SESSION_ID] invalid session id (_ssl.c:1000).
...
```

the following happens in the OpenWrt log (also a bunch of identical messages):
```
...
Sat Aug 17 16:34:42 2024 daemon.info youtubeUnblock[12787]: Target SNI detected: rr2---sn-a5ou5-ucfe.googlevideo.com
Sat Aug 17 16:34:42 2024 daemon.info youtubeUnblock[12787]: Packet split in portion 2 515
Sat Aug 17 16:34:42 2024 daemon.info youtubeUnblock[12787]: Packet split in portion 144 371
...
```

I run `youtubeUnblock` with following arguments: `--faking-strategy=ttl --faking-ttl=7`.

**Comment by Waujito** [comment]
Try —fake-sni=0 flag

**Comment by proninyaroslav** [comment]
Yes, it works. I'll have to use VPN because I can't turn off this flag due to the android tv😄

**Comment by Waujito** [comment]
The issue doesn't solved. Can you try dev branch with two more faking strategies? Binaries are here: https://github.com/Waujito/youtubeUnblock/actions/runs/10431361822

**Comment by proninyaroslav** [comment]
First of all, I had to lower the faking ttl value from 7 to 6 so that android tv would start playing the video immediately, and not with a delay. Before the update, it worked on 7 without delays. There are also changes with `yt-dlp`, it says a timeout error:

```
[download] Got error: HTTPSConnectionPool(host='rr2---sn-a5ou5-ucfe.googlevideo.com', port=443): Read timed out. (read timeout=20.0). Retrying (1/10)...
```

**Comment by Waujito** [comment]
Have you tried --faking-strategy=pastseq and --faking-strategy=tcp_check

**Comment by proninyaroslav** [comment]
`pastseq` seems works (tv & yt-dlp)

**Comment by Waujito** [comment]
So, if everything works, I'm closing this.
